app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -81,7 +81,6 @@
     justify-content: center;
     align-items: center;
     overflow: hidden;
-    background-image: url('/assets/onboarding-background-white.png');
```

Comment:
This image is specific to DEV since it contains the DEV badges and should not be hardcoded in our codebase for Forem. However, I just want to make a note that when I remove this there is a flash before the asset from the site config shows up on the page. I'm not sure whats the alternate solution to this?

Reply:
Maybe @ludwiczakpawel wants to weigh in on this?